src/labels/CustomLabels.labels

```diff
@@ -6181,6 +6213,46 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <!-- {0} is a string. Name of the template that was created. -->
         <value>The "{0}" template was updated.</value>
     </labels>
+    <labels>
+        <fullName>geUtilJWTTestEmptyValue</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Unit test error: test with empty value</shortDescription>
```

Comment:
@ptewson-sfdo @vmartinov-sforg @psadie-sfdo I defer to you all for clear verbiage around the unit test labels.

Reply:
Please don a hat of a brand new hire who runs into these unit tests and make sure the person would be able to know what to check if they need to fix anything.

Reply:
@melissabarber I don't think this label is actually being used - @voduyemi should we remove it? Or did I just search wrong?

Reply:
@ptewson-sfdo You searched right! I was supposed to remove it.